### PR TITLE
test(http-core): fix interceptor log test

### PR DESCRIPTION
**Pull request by InteractiveTimmy**
# Pull Request

## Description

Update the interceptor log test to validate if verbose networking logging is either enabled or disabled before running the test.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Ran testing against http-core to confirm change works.

**Test Configuration**:
* Node/Browser Version v8.16.0
* NPM Version v6.4.1

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by taymoork2** [comment]
Is there a way to fake the env being true, rather than skipping the test all together?

File: packages/node_modules/@webex/http-core/test/integration/spec/interceptor.js [hunk 1]
```diff
@@ -22,7 +22,16 @@ describe('http-core', () => {
       })));
 
     describe('logOptions', () => {
-      it('calls logger plugin', () => {
+      it('calls logger plugin if enabled', () => {
+        // this if statement catches and skips test if verbose network logging
+        // is not enabled.
+        if (!process.env.ENABLE_VERBOSE_NETWORK_LOGGING) {
```
Comment (adamweeks) [inline packages/node_modules/@webex/http-core/test/integration/spec/interceptor.js]: Like @taymoork2 suggested, should we set the logging to be true manually to validate the test?

File: packages/node_modules/@webex/http-core/test/integration/spec/interceptor.js [hunk 2]
```diff
@@ -22,15 +22,33 @@ describe('http-core', () => {
       })));
 
     describe('logOptions', () => {
-      it('calls logger plugin', () => {
+      it('calls logger plugin if enabled', () => {
```
Comment (adamweeks) [inline packages/node_modules/@webex/http-core/test/integration/spec/interceptor.js]: In order to achieve the ["Arrange, Act, Assert"](https://defragdev.com/blog/?p=783), I'd prefer to see the setup of the env vars in a `beforeEach` block. Then returning the env var back in an `afterEach` block.

File: packages/node_modules/@webex/http-core/test/integration/spec/interceptor.js [hunk 3]
```diff
@@ -22,15 +22,33 @@ describe('http-core', () => {
...
         const spy = sinon.spy(webex.logger, 'info');
 
         return webex.request({
           service: 'hydra',
           resource: 'people/me'
         }).then(() => {
+          // toggle to disabled if originally disabled
+          toggleVNL();
           assert.called(spy);
         }).catch(() => {
```
Comment (adamweeks) [inline packages/node_modules/@webex/http-core/test/integration/spec/interceptor.js]: We shouldn't need both a `.catch` and `.then` here. The test should follow the expected path of one or the other.

**Pull request merged**